mr/src/test/java/org/apache/iceberg/mr/hive/TestDeserializer.java

```diff
@@ -155,13 +155,13 @@ public void testListDeserialize() {
     Object[] data = new Object[] {new Object[] {new LongWritable(1L)}};
     Record actual = deserializer.deserialize(data);
 
-    Assert.assertEquals(expected, actual);
+    assertThat(actual).isEqualTo(expected);
   }
 
   @Test
   public void testDeserializeEverySupportedType() {
-    Assume.assumeFalse(
-        "No test yet for Hive3 (Date/Timestamp creation)", HiveVersion.min(HiveVersion.HIVE_3));
+    assumeThat(HiveVersion.min(HiveVersion.HIVE_3))
+        .as("No test yet for Hive3 (Date/Timestamp creation)");
```

Comment:
this needs a `.isFalse()` at the end, otherwise it wouldn't anything

Reply:
Yes. I missed that. Thank you for noticing.